plugins/liferay-maven-plugin/src/test/resources/projects/servicebuilder/builder/builder-portlet/src/main/resources/META-INF/base-spring.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>
```

Comment:
these resources are generated as are not needed in the test resources, only the minimal test project should be in the resources directory